workflows/kallisto-quant/run-kallisto.nf

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+// run parameters
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.index_dir = 'kallisto_index'
+params.index_name = 'cdna_k31'
+params.annotation_dir = 'annotation'
+params.t2g = 'Homo_sapiens.ensembl.100.tx2gene.tsv'
+params.mitolist = 'Homo_sapiens.ensembl.100.mitogenes.txt'
```

Comment:
This doesn't get used anywhere yet, correct?

Reply:
Correct.  It might never get used, I suppose.